config/default.json

```diff
@@ -84,7 +84,7 @@
     "testingSupport": false,
     "countyCourtJudgment": false,
     "fullAdmission": false,
-    "partialAdmission": false,
+    "partialAdmission": true,
```

Comment:
this will be changed before merge

Reply:
it is already enabled in `development.json` so it is on by default locally you shouldn't need to change this

Reply:
ok